packages/connector/src/index.ts

```diff
@@ -9,24 +10,20 @@ export const defaultSnapOrigin = 'npm:@rarimo/rarime';
 
 export const enableSnap = async (
   snapOrigin?: string,
-  version = '0.4.x',
+  version = versionJson.version,
```

Comment:
Not need to force user to install every single patch. Let's add a separate field to `version.json` with the supported semver range